src/data.cpp

```diff
@@ -1596,15 +1596,18 @@ double pyne::decay_const(std::string nuc) {
 // Half-life data
 //
 double pyne::half_life(int nuc) {
-    std::vector<double> result = data_access<double, level_data>(nuc, 0.0,
-    DBL_MAX, offsetof(level_data, half_life), level_data_lvl_map);
-    if (result.size() == 1) {
-        return result[0];
-    }
-    return 1.0/0.0;
+  std::vector<double> result = data_access<double, level_data>(
+      nuc, 0.0, DBL_MAX, offsetof(level_data, half_life), level_data_lvl_map);
+  if (result.size() == 1) {
+    return result[0];
+  }
+#ifndef _WIN32
```

Comment:
We should return the same result, regardless of platform.

Reply:
And we should also make sure that this result is not expected in other places!

Reply:
maybe we could disable decay for windows ?

Reply:
Is there no way to return some equivalent of infinity in windows?